content/root/glossary.textile

```diff
@@ -0,0 +1,212 @@
+
+h2(#pub-sub). Pub/Sub
+
+Publish-subscribe (or pub/sub), using Ably terms, is a pattern of information exchange where a publisher does not need to interact directly with, or even know the existence or identities of, any subscribers. Conversely, a subscriber can gain access to published information without interacting directly with the publisher. The ratio of publishers to subscribers can be M:N, across any number of channels.
+
+h2(#publisher). Publisher
+
+A publisher is an entity that sends messages on one or more channels.
+
+h2(#push-notification). Push notifications
+
+Push notifications are messages that are pushed to a device's user interface such as a visual notification. Ably can deliver push notifications using platforms such as Apple's Push Notification service or Google's Firebase Cloud Messaging service.
+
+See the "push notification documentation":/realtime/push for further information.
+
+h2(#queues). Queues
+
+Reactor Queues provide a service that distributes messages between subscribers, so that each message is only received by one subscriber. This model is referred to as "competing consumers" and is different to the pub/sub model where each message is delivered to every subscriber. Ably uses integration rules to publish messages received in a channel into a queue.
+
+See the "queues documentation":/general/queues for further information.
+
+h2(#realtime-sdk). Realtime client library SDK
+
+The realtime client library SDKs are used to add long-lived realtime connections to Ably within applications.
+
+See the "SDK downloads page":https://ably.com/download for further information on SDKs and the available languages.
+
+h2(#rest-sdk). REST client library SDK
+
+The REST client library SDKs provide an API to interact with the Ably REST (HTTP) interface.
+
+See the "SDK downloads page":https://ably.com/download for further information on SDKs and the available languages.
+
+h2(#rewind). Rewind
+
+Rewind is a feature that enables a client to subscribe to a channel from a position earlier than the current point in time. The @rewind@ parameter can specify an earlier position as a time interval or set number of messages. When a client first attaches to a channel using rewind, all messages from that earlier position are replayed to them.
+
+See the "Rewind documentation":/realtime/channels/channel-parameters/rewind for further information.
+
+h2(#sse). SSE
+
+Server Sent Events (SSE) is an HTTP-based protocol for streaming events from a server to a client. In the context of Ably, SSE can be used to establish a realtime subscription to a set of channels.
+
+See the "SSE documentation":/sse for further information on using SSE with Ably.
+
+h2(#stomp). STOMP
+
+Simple (or Streaming) Text Orientated Messaging Protocol (STOMP) is a protocol that can be used to retrieve messages from a queue.
+
+See "Wikipedia":https://en.wikipedia.org/wiki/Streaming_Text_Oriented_Messaging_Protocol for further information.
```

Comment:
I know it's not very specific to STOMP, but is it worth linking to https://ably.com/documentation/general/queues here as well?

Reply:
Linked so it mirrors AMQP.